### DISCUSSION TITLE
Can anyone help me?

### DISCUSSION THREAD

**Discussion by AssaultStars** [tltneon/lgsl#163] [Q&A]
Hey my friends, first thank you for your great work, this must be recognized!

Undzwar I have installed everything, however, it shows my server with 0/0 players or offline.

My server appears for example at www.game-state.com as online, so are all ports open or not?

Here is a link to the LGSL
https://h574355.web298.dogado.net/lgsl/
Why does it show offline for me?

I hope someone can help me

The other question would be, can you make a similar site like gametracker or game-state with this system?

Thanks

**Comment by BaxAndrei** [comment]
Admin page then check if fsockets is yes
![image](https://github.com/tltneon/lgsl/assets/12484505/2b5b7839-b38d-4039-858d-f44da0944d0d)

**Comment by AssaultStars** [comment]
fsockets is yes sir.
Other ideas? ;(

**Reply by AssaultStars** [reply]
it say some games need bz2 php.ini .. for example for cs 1.6 my server. How to do that?

**Reply by ZekyTheWolf** [reply]
> it say some games need bz2 php.ini .. for example for cs 1.6 my server. How to do that?

Depends, if you have own server you need to install it e.g. `apt instal php8.2-bz2` or enable it inside php.ini.

If on provider, that should be enabled by defaul.

**Comment by BaxAndrei** [comment]
![image](https://github.com/tltneon/lgsl/assets/12484505/4462bf79-b47a-40b1-99cf-0a697bcde5b8)
I see there your gameport is closed

**Reply by AssaultStars** [reply]
I have spoken with the game server hoster. He said that all ports are free. What I also don't understand is, on Gamestate and other platforms it is shown online. Except for the LGSL. I am unquestionably overwhelmed. He say ill try maybe Query Ports like 27016/17 but nothing do. 
and this is my linke: https://h574355.web298.dogado.net/lgsl/

https://www.game-state.com/147.189.175.60:27015/
https://tsarvar.com/de/servers/counter-strike-1.6/147.189.175.60:27015

**Comment by ZekyTheWolf** [comment]
1. Check webserver / php error logs
2. Check if you have all required dependancy (bz2, sockets, ...)
But this looks like you dont have something enabled or missing dependancy package...

**Comment by AssaultStars** [comment]
I'll try many ports. And look for errors but nothing. when i install it says YES bz2 etc for everything. Success. Are probably all necessary ports open. I do not understand this. Can someone maybe help me if I send the credentials? Someone give me his contact details please?
http://h574355.web298.dogado.net/lgsl/

Email is: info@assault-stars.de

Edit: i've try more and other server ips and games. But show all offline.

**Reply by Rosenstein** [reply]
There is no line FSOCKOPEN check on your installation page.

**Comment by AssaultStars** [comment]
![grafik](https://github.com/tltneon/lgsl/assets/146896703/fba93c8c-d6d5-4e83-8998-b94756289301)
![Screenshot 2023-10-08 225015](https://github.com/tltneon/lgsl/assets/146896703/e943664b-c885-463b-bd6f-26bec478abc5)
its open or not?
